homeassistant/components/geocaching/sensor.py

```diff
@@ -46,7 +45,7 @@ class GeocachingSensorEntityDescription(
         measurement="find_count",
         icon="mdi:notebook-edit-outline",
         native_unit_of_measurement="caches",
-        default_enabled=True,
+        entity_registry_enabled_default=False,
```

Comment:
@frenck I suspect a copy paste bug? This one should be True. Otherwise adding this integration will start without sensors.

Reply:
Yes :) true is the default and thus is can be removed in that case